azurelinuxagent/ga/persist_firewall_rules.py

```diff
@@ -199,8 +199,8 @@ def _setup_network_setup_service(self):
 
             # Create unit file with default values
             self.__set_service_unit_file()
-            # Reload systemd configurations when we setup the service for the first time to avoid systemctl warnings
-            self.__reload_systemd_conf()
+            # Initially systemd daemon-reload added here to prevent systemctl warnings after unit file changes. But that warning is local to that service and it doesn't affect other services
```

Comment:
Minor: I would rephrase the comment to say that after creating the service, systemctl may issue a warning when checking the service, and daemon-reload should not be used to clear the warning, since it can affect other services

Reply:
`after creating the service, systemctl may issue a warning when checking the service, `

This statement not true, when creating the service, it will not give warning, it's only issue a warning when we modify the service

`root@ub20euap:/lib/systemd/system# vi hello.service

root@ub20euap:/lib/systemd/system# systemctl status hello
● hello.service - Hello World Service
     Loaded: loaded (/lib/systemd/system/hello.service; static; vendor preset: enabled)
     Active: inactive (dead)
`